.github/workflows/publish-sample.data.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  workflow_dispatch:
+    inputs:
+      current_sample_branch:
+        description: Publish new spartacus sample data to standard Spartacus ccv2 repository
+        default: release/2105/next
+        required: true
+      previous_sample_branch:
+        description: Publish previous version spartacus sample data to standard Spartacus ccv2 repository
+        default: release/2105/4dot4
+        required: true
+
+name: Publish Spartacus Sample Data to Spartacus CCv2 repository
+jobs:
+  publish_sample_data: 
+    name: Publishing Spartacus Sample Data
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: create release with sample data assets
+        env: 
+          SAMPLE_DATA_NEW: ${{ github.event.inputs.current_sample_branch }}
+          SAMPLE_DATA_OLD: ${{ github.event.inputs.previous_sample_branch }}
+          GHT_USER: ${{ secrets.GHT_USER }}
```

Comment:
Small stuff, why did you go with GHT instead of GH? GH is more commonly known for Github. Does the T have to do with the Technical user?